README.Rmd

```diff
@@ -31,7 +31,98 @@ utilizes some of its code base.
 
 ### Windows
 
-### Linux
+Start your preferred R environment and run the following R commands from within the ```rSharp``` folder to install the package using prebuilt binaries
+
+```
+install.packages("devtools")
+devtools::install_github("https://github.com/Open-Systems-Pharmacology/rSharp")
+```
+
+### Ubuntu
+
+Run the following commands to get set up to install and use the R package using prebuilt binaries
+
+```
+sudo apt-get install dotnet-runtime-8.0 libcurl4-openssl-dev libssl-dev libxml2-dev 
+sudo apt-get install libfontconfig1-dev libharfbuzz-dev libfribidi-dev
+sudo apt-get install libfreetype6-dev libpng-dev libtiff5-dev libjpeg-dev
+sudo ln -s /usr/lib/x86_64-linux-gnu/libdl.so.2 /usr/lib/x86_64-linux-gnu/libdl.so
+```
+Symlink libdl.so.2 to libdl.so
+
+```
+sudo ln -s /usr/lib/x86_64-linux-gnu/libdl.so.2 /usr/lib/x86_64-linux-gnu/libdl.so
+```
+
+Start your preferred R environment and run the following R commands from within the ```rSharp``` folder
+
+```
+install.packages("devtools")
+devtools::install_github("https://github.com/Open-Systems-Pharmacology/rSharp")
+```
+
+## Build
+
+### Windows
+
+Download and install R https://cran.r-project.org/bin/windows/base/old/4.3.1/
```

Comment:
We need a way to remove this specific version dependency. It's important for building the binaries to have the header files, but we should not specify what version. I'll open an issue